Framework/PythonInterface/plugins/functions/BivariateGaussian.py

```diff
@@ -7,7 +7,10 @@
 from __future__ import (absolute_import, division, print_function)
 import numpy as np
 from mantid.api import IFunction1D, FunctionFactory
-from matplotlib.mlab import bivariate_normal
```

Comment:
Should we find a better solution to this?

`matplotlib.mlab.bivariate_normal` is deprecated and requiring Matplotlib for a fitting function seems a bit backwards.

Reply:
Actually it's a pretty small function, maybe we just copy the [source](https://matplotlib.org/2.2.4/_modules/matplotlib/mlab.html#bivariate_normal).